recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt

```diff
@@ -126,6 +127,7 @@ class RecyclerViewFastScroller @JvmOverloads constructor(
         const val DEFAULT_ANIM_DURATION: Long = 100
         const val DEFAULT_POPUP_VISIBILITY_DURATION = 200L
         const val hasEmptyItemDecorator: Boolean = true
```

Comment:
```suggestion
        const val DEFAULT_HAS_EMPTY_ITEM_DECORATOR: Boolean = true
```

Reply:
I didn't even add this part?

Reply:
agreed. let me make a different issue on the same.